src/keyboard/LayerPicker.tsx

```diff
@@ -183,7 +183,7 @@ export const LayerPicker = ({
           <button
             type="button"
             disabled={!canAdd}
-            className="hover:text-primary-content ml-1 hover:bg-primary rounded-sm disabled:text-gray-500 disabled:hover:bg-base-300 disabled:cursor-not-allowed"
```

Comment:
Any reason `ml-1` was removed? just ends up being a little gap between the two buttons

Reply:
yes, i intended to place the gap to its parent but seems like i forgot to do so. i updated this PR with the changes.

```diff
-       <div className="grid grid-cols-[1fr_auto_auto] items-center">
+       <div className="grid grid-cols-[1fr_auto_auto] items-center gap-1">
```